tests/testautotls.nim

```diff
@@ -0,0 +1,45 @@
+{.used.}
```

Comment:
- what is request rate limit for `LetsEncryptURLStaging` api?
- if we are doing these request for each commit in CI for each developer are we going to overdo it?
   - consider 18 jobs (os x nim version x architecture)  x number of commits daily say 100 
  - if so, can we run daily tests with additional environment variable (eg. `DAILY_TEST='yes'`) and run api tests only if `let isDaily = getEnv("DAILY_TEST", "no") == "yes"`

Reply:
I wanted to discuss this but did not get many replies: https://discord.com/channels/864066763682218004/1113778107359436821/1375151097135501364

Reply:
also consider implementing this as the last piece, or disabling it if implemented earlier, so we can have CI check while developing. :)

Reply:
if test is run locally, by default, it will definitely reach limit, i was ruining tests more then this limit

Reply:
I'd say unlikely but how it is set right now the only way to run the autotls tests is intentionally through `nimble testautotls` (`nimble testall` or `nimble test` will not trigger autotls`

https://letsencrypt.org/docs/staging-environment/#rate-limits